Makefile

```diff
@@ -0,0 +1,32 @@
+all: reinit_db configure init accounts assets start
```

Comment:
I still think it'd be nice to prefix these make commands with `docker-ledger` or something to be clear that it's running the ledgers on docker.